libraries/Microsoft.Bot.Builder.AI.QnA/Models/AnswerSpanResponse.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    public class AnswerSpanResponse
```

Comment:
Comment required

Reply:
Done.

---
In reply to: [430552196](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430552196) [](ancestors = 430552196)